### PR TITLE
Fix inconsistency in properties names (#54)

### PR DESCRIPTION
**Pull request by sdelatorrep**
They should start with lowercase to keep consistency among all properties.

### CODE REVIEW

**Comment by sdelatorrep** [comment]
@redmitry, I can't add you as a reviewer, but, as you were the first noticing this inconsistency, you may want to take a look. Thanks! :)

**Review by mbaudis — APPROVED** [review]

File: beacon.yaml [hunk 1]
```diff
@@ -1245,14 +1245,14 @@ components:
...
+        info:
           type: string
           example: 'beacon-info-v2.0.0-draft.3'
-        Dataset:
+        dataset:
           type: string
           example:
             'beacon-dataset-v2.0.0-draft.3'
-        FilterTerm:
+        filterTerm:
```
Comment (mbaudis) [inline beacon.yaml]: filterTerm => filteringTerm

**Review by mbaudis — CHANGES_REQUESTED** [review]
See note on line 1255; should be "filteringTerm". Otherwise ok (well. I did approve beffore ...).

File: beacon.yaml (hunk 1 above)
Comment (sdelatorrep) [reply beacon.yaml]: Done!

**Review by mbaudis — APPROVED** [review]

**Pull request merged**